docs/niscope/functions.rst

```diff
@@ -1224,29 +1233,54 @@ niscope.Session methods
 
 
     :type num_samples: int
-    :param timeout:
+    :param fetch_relative_to:
 
 
-        The time to wait in seconds for data to be acquired; using 0 for this parameter tells NI-SCOPE to fetch whatever is currently available. Using -1 for this parameter implies infinite timeout.
+        Position to start fetching within one record. If not set, use value of NISCOPE_ATTR_FETCH_RELATIVE_TO
 
         
 
 
-    :type timeout: float
+    :type fetch_relative_to: :py:data:`niscope.FetchRelativeTo`
+    :param fetch_offet:
 
-    :rtype: tuple (wfm, wfm_info)
 
-        WHERE
+        Offset in samples to start fetching data within each record. The offset is applied relative to fetch_relative_to. The offset can be positive or negative. If not set, use value of NISCOPE_ATTR_FETCH_OFFSET
 
-        wfm (list of float): 
+        
 
 
-            Returns an array whose length is the **numSamples** times number of waveforms. Call :py:meth:`niscope.Session._actual_num_wfms` to determine the number of waveforms.
+    :type fetch_offet: int
+    :param fetch_record_number:
 
-            
 
+        Zero-based index of the first record to fetch.  Use fetch__num_records to set the number of records to fetch. If not set, use value of NISCOPE_ATTR_RECORD_NUMBER
```

Comment:
two underscores (type)
If you call out the attribute, use its python name. Applies elsewhere.
"If not set, use value of record_number property"

Reply:
Two underscores fixed.

Anything with <DRIVER>_ATTR_<C_NAME> should be replaced with a link to session.<python_name>. I need to figure out why these aren't.